notebooks/SpecvizExample.ipynb

```diff
@@ -75,17 +76,9 @@
    "source": [
     "## Load a file to Specviz\n",
     "\n",
-    "Here we download a JWST spectrum from [MAST](https://masttest.stsci.edu/) using `astroquery`. \n",
-    "By default the downloaded file goes to your temp directory, and thus may eventually be \n",
-    "garbage collected (deleted) by your system. If you would like to have the file permanently, \n",
-    "simply uncomment the second line below and provide the directory path on your system where \n",
-    "you would like the file stored.\n",
+    "We will download an example JWST spectrum using `astropy`. The downloaded file can be found in the '/Users/username/.astropy/cache/download/' folder.\n",
```

Comment:
Should we advertise the current default download destination path in `astropy.utils.data.download_file` as part of these docs? Maybe [a link](https://docs.astropy.org/en/stable/api/astropy.utils.data.download_file.html) to the docs on that function would be better.